lib/ceased_member_row.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class CeasedMemberRow < Scraped::HTML
+  field :source do
+    noko.css('a @href').text
+  end
+
+  field :name do
+    noko.css('a').text
+  end
+
+  field :party do
+    # The layout is inconsistent. Scraping this way
+    # we always get the party name for each row
+    noko.xpath('text()')[0..1].text.split('.').last.tidy
```

Comment:
Silvana Boneva's row is slightly different from the others. In the final three rows, the party ID is either in the first or second text nodes. I've added a test for Silvana Boneva.